console/BUILD

```diff
@@ -101,6 +101,18 @@ linux_packages(
 )
 
 
+rpm_package(
+    name = "deploy-rpm",
+    package_name = "grakn-core-console",
+    installation_dir = "/opt/grakn/core/console/",
+    version_file = "//:VERSION",
+    spec_file = "//dependencies/distribution/rpm:grakn-core-console.spec",
+    target = ":console-binary",
+    empty_dirs = [
+         "./opt/grakn/core/console/services/lib/",
```

Comment:
`opt/...`

Reply:
@lolski fixed in https://github.com/graknlabs/grakn/pull/4777/commits/1456af49e17ee353ec93c7b05a844d2ff421ab7c